src/foam/comics/v2/userfeedback/UserFeedbackStatus.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.comics.v2.userfeedback',
+  name: 'UserFeedbackStatus',
+  documentation: `
+    Functional statuses which highlight different types of feedback
+    the client user is interested in
+  `,
+
+  values: [
```

Comment:
I think WARNING would be useful as well.

Reply:
Done.